nixos/modules/tasks/filesystems/bcachefs.nix

```diff
@@ -41,10 +41,17 @@ let
       # also, implement automatic waiting for the constituent devices when that happens
       # bcachefs does not support mounting devices with colons in the path, ergo we don't (see #49671)
       firstDevice = head (splitString ":" fs.device);
+      withClevis = config.boot.initrd.clevis.enable && (hasAttr firstDevice config.boot.initrd.clevis.devices);
     in
-      ''
-        tryUnlock ${name} ${firstDevice}
-      '';
+    if withClevis then ''
+      if clevis decrypt < /etc/clevis/${firstDevice}.jwe | bcachefs unlock ${firstDevice}; then
+        printf "unlocked ${name} using clevis\n"; else
```

Comment:
This is weird formatting. `else` should just be on  a separate line at the same indentation level as `if`.